mne/viz/_3d.py

```diff
@@ -2584,15 +2584,15 @@ def snapshot_brain_montage(fig, montage, hide_sensors=True):
     im : array, shape (m, n, 3)
         The screenshot of the current scene view
     """
-    from ..channels import Montage, DigMontage
+    from ..channels import DigMontage
     from .. import Info
     # Update the backend
     from .backends.renderer import _Renderer
 
     if fig is None:
         raise ValueError('The figure must have a scene')
-    if isinstance(montage, (Montage, DigMontage)):
```

Comment:
This code was only referring to `DigMontage` since `Montage` has no `dig_ch_pos`. (I guess that this is how the code was intended. see the error raising has no reference to Montage)